pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -248,6 +248,11 @@ type V2RegistriesConf struct {
 	// potentially use all unqualified-search registries
 	ShortNameMode string `toml:"short-name-mode"`
 
+	// AdditionalLayerStoreAuthHelper is a helper binary that receives
+	// registry credentials pass them to Additional Layer Store for
+	// registry authentication.
+	AdditionalLayerStoreAuthHelper string `toml:"additional-layer-store-auth-helper"`
```

Comment:
At least `.Nonempty()` and `.updateWithConfigurationFrom()` also need updating.

Reply:
Fixed updateWithConfigurationFrom. I guess nothing to be updated in `Nonempty` because it is  a string field?

Reply:
Added a test.